db/schema.rb

```diff
@@ -11,304 +10,294 @@
-    t.string   "school",                limit: 255
-    t.string   "term",                  limit: 255
-    t.integer  "character_sum",         limit: 4,     default: 0
-    t.integer  "view_sum",              limit: 8,     default: 0
-    t.integer  "user_count",            limit: 4,     default: 0
-    t.integer  "article_count",         limit: 4,     default: 0
-    t.integer  "revision_count",        limit: 4,     default: 0
-    t.string   "slug",                  limit: 255
-    t.string   "subject",               limit: 255
-    t.integer  "expected_students",     limit: 4
+    t.string   "school"
+    t.string   "term"
+    t.integer  "character_sum",                       default: 0
+    t.bigint   "view_sum",                            default: 0
+    t.integer  "user_count",                          default: 0
+    t.integer  "article_count",                       default: 0
+    t.integer  "revision_count",                      default: 0
+    t.string   "slug"
+    t.string   "subject"
+    t.integer  "expected_students"
     t.text     "description",           limit: 65535
     t.boolean  "submitted",                           default: false
-    t.string   "passcode",              limit: 255
+    t.string   "passcode"
     t.datetime "timeline_start"
     t.datetime "timeline_end"
     t.string   "day_exceptions",        limit: 2000,  default: ""
-    t.string   "weekdays",              limit: 255,   default: "0000000"
-    t.integer  "new_article_count",     limit: 4,     default: 0
+    t.string   "weekdays",                            default: "0000000"
+    t.integer  "new_article_count",                   default: 0
     t.boolean  "no_day_exceptions",                   default: false
-    t.integer  "trained_count",         limit: 4,     default: 0
-    t.integer  "cloned_status",         limit: 4
-    t.string   "type",                  limit: 255,   default: "ClassroomProgramCourse"
-    t.integer  "upload_count",          limit: 4,     default: 0
-    t.integer  "uploads_in_use_count",  limit: 4,     default: 0
-    t.integer  "upload_usages_count",   limit: 4,     default: 0
-    t.string   "syllabus_file_name",    limit: 255
-    t.string   "syllabus_content_type", limit: 255
-    t.integer  "syllabus_file_size",    limit: 4
+    t.integer  "trained_count",                       default: 0
+    t.integer  "cloned_status"
+    t.string   "type",                                default: "ClassroomProgramCourse"
+    t.integer  "upload_count",                        default: 0
+    t.integer  "uploads_in_use_count",                default: 0
+    t.integer  "upload_usages_count",                 default: 0
+    t.string   "syllabus_file_name"
+    t.string   "syllabus_content_type"
+    t.integer  "syllabus_file_size"
     t.datetime "syllabus_updated_at"
-    t.integer  "home_wiki_id",          limit: 4
+    t.integer  "home_wiki_id"
+    t.string   "time_zone",                           default: "UTC"
```

Comment:
So you can sort it out by reverting to the original version, and then just manually updating those two lines.